tools/boot/install-boot.sh

```diff
@@ -60,6 +60,10 @@ make_esp_file() {
     mkdir -p "${stagedir}/EFI/BOOT"
     efibootname=$(get_uefi_bootname)
     cp "${loader}" "${stagedir}/EFI/BOOT/${efibootname}.efi"
+    loader=$(echo ${loader} | sed s,.efi,-ia32.efi,)
```

Comment:
. isn't escaped, you probably should anchor it, and all of this can be just done in a plain shell expansion with `${loader%.efi}-ia32.efi`